PRIVACY.md

```diff
@@ -109,6 +109,7 @@ Semgrep reports data that indicate how useful a run is for the end user; e.g.
 - Number of ignored findings
 - Pseudoanonymized hashes of the rule definitions that yield findings
 - The [Semgrep features used](#feature-usage) during the scan
+- File extensions of unscanned files
```

Comment:
there's also a sample JSON blob code snippet, and a detailed breakdown table in this file; could you update both?

Reply:
Just updated. To make the changes more straightforward, I moved the `unsupportedExts` key under `values`. Let me know if you think this doesn't make sense and I can put it back---I thought it seemed more consistent with the existing metrics though.